orderer/common/msgprocessor/maintenancefilter.go

```diff
@@ -199,3 +204,59 @@ func (mf *MaintenanceFilter) ensureConsensusTypeChangeOnly(configEnvelope *cb.Co
 
 	return nil
 }
+
+func validateBFTMetadataOptions(selfID uint64, options *smartbft.Options) (types.Configuration, error) {
```

Comment:
I did it since trying to reuse the function by making it public causes an import cycle which is not allowed. 
The cycle is between:
imports github.com/hyperledger/fabric/orderer/consensus/smartbft
imports github.com/hyperledger/fabric/orderer/common/msgprocessor